docs/content/deployment/guides/dask.mdx

```diff
@@ -25,24 +25,16 @@ First, run `pip install dagster-dask`.
 
 Then, you'll need to add the dask executor to a **`ModeDefinition`** on your pipeline:
 
-```python file=/deploying/dask_hello_world.py
-from dagster import ModeDefinition, default_executors, fs_io_manager, pipeline, solid
-from dagster_dask import dask_executor
```

Comment:
I think it's helpful to retain the imports.  Without them, someone copy/pasting the code will need to do additional investigation to get it working.